.github/workflows/gh_release.yml

```diff
@@ -0,0 +1,59 @@
+# This workflow will install Python dependencies, run pre-release script, and
+# then bundle a github release
+
+name: Release generator
+
+on:
+  pull_request:
+    types:
+      - closed
+
+jobs:
+  context:
+    if: github.base_ref == 'master' && github.event.pull_request.merged && contains( github.event.pull_request.labels.*.name, 'release')
```

Comment:
I think it is _probably_ preferable. Keep in mind that the label is added automatically by the user tool.

Reply:
I'm open to being convinced against using a label though.

Reply:
Label tag sound good because we currently label all PRs except for releases. We just need to make sure we have a standard release label across all repos.

Reply:
I like the the label idea! Especially because the user doesn't have to remember to add it since the tool does it